lib/_emerge/BinpkgFetcher.py

```diff
@@ -90,6 +91,9 @@ def _start(self):
 		bintree = pkg.root_config.trees["bintree"]
 		settings = bintree.settings
 		pkg_path = self.pkg_path
+		binpkg_format = settings.get("BINPKG_FORMAT", "xpak")
```

Comment:
I think long-term we'd want to support fetching both formats, possibly detecting the correct format from binhost's metadata.

Reply:
added todo

Reply:
I'd like for it to support fetching of both formats from the beginning, so that there's a smooth upgrade path.